SyntaxError: The given name 'foo' is already used in another export [selective_export_clashing_exports_3.sts:21:10]
